internal/mptrie/mptrie_test.go

```diff
@@ -863,29 +870,39 @@ func (s *trieStoreMock) GetNode(nodePtr []byte) (TrieNode, error) {
 	key := base64.StdEncoding.EncodeToString(nodePtr)
 	nodeBytes, ok := s.persistNodes[key]
 	if !ok {
-		nodeBytes = s.inMemoryNodes[key]
+		nodeBytes, ok = s.inMemoryNodes[key]
+		if !ok {
+			fmt.Printf("can't get node for ptr %s\n", key)
```

Comment:
why `fmt.Printf`? Moreover, do we meed this log message at all?

Reply:
I will return not found error here.